### PR TITLE
Update #442

### PR DESCRIPTION
**Pull request by dharvey17**
@ptatian ready for review

In my previous note, I'd confused creating the HUD files with this step (hence the comment about the Update-#)

### CODE REVIEW

File: Prog/Updates/Update_Sec8mf_2023_11.sas
Comment (ptatian) [inline Prog/Updates/Update_Sec8mf_2023_11.sas:1]: This program doesn't have any parameters filled in. But we don't need to run earlier downloads if we missed them earlier. We can just run the 2024_01 file. So we can delete this program.

File: Prog/Updates/Update_REAC_2023_11.sas
Comment (ptatian) [inline Prog/Updates/Update_REAC_2023_11.sas:1]: In the listing output for the REAC program, there is a list of "Nonmatching REAC records" Can you please check to see if any of these are projects in the Catalog? If so, you can update the Macros\Update_REAC_init.sas program to add the correspondence between the REAC_ID and the Catalog NLIHC_ID.

File: Prog/Updates/Update_Sec8mf_2024_01.sas
Comment (ptatian) [inline Prog/Updates/Update_Sec8mf_2024_01.sas:1]: Looks good!

**Review by ptatian — CHANGES_REQUESTED** [review]
Hi Donovan. Looks good. Asking that you delete the Update_Sec8mf_2023_11.sas program and see if any of the nonmatching projects in the REAC data are in the Catalog. Thanks!

**Comment by dharvey17** [comment]
List of non-matching properties

- The Foundry Lofts
- City Market at O Street
- The Lexington at Market Square
- Arbor View Apartments
- Indigo Apartments 
- The Griffin Apartments
- Zen Apollo

**Comment by dharvey17** [comment]
Hi Peter, I thought I tagged you in case you wanted to take a look at the changes you flagged.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ptatian** [comment]
Yes, that's fine. I just didn't know if you were finished. I'll review now.

**Review by ptatian — APPROVED** [review]
Thanks, @dharvey17 OK to merge and do the final batch submit.

**Pull request merged**